packages/editor/src/plugins/image-gallery/components/static/static-lightbox-mobile.tsx

```diff
@@ -0,0 +1,102 @@
+import {
+  faChevronLeft,
+  faChevronRight,
+} from '@fortawesome/free-solid-svg-icons'
+import { FaIcon } from '@serlo/frontend/src/components/fa-icon'
+import { lazy, Suspense, UIEvent } from 'react'
+
+import { StaticLightboxCarousel } from './static-lightbox-carousel'
+import { GridImage } from '../../types'
+import { useInstanceData } from '@/contexts/instance-context'
+import { cn } from '@/helper/cn'
+
+const ModalWithCloseButton = lazy(() =>
+  import('@/components/modal-with-close-button').then((module) => ({
+    default: module.ModalWithCloseButton,
+  }))
+)
+
+interface StaticLightboxMobileProps {
+  images: GridImage[]
+  isOpen: boolean
+  currentImageIndex: number
+  setIsOpen: (isOpen: boolean) => void
+  setCurrentImageIndex: (index: number | null) => void
+}
+
+export function StaticLightboxMobile({
+  images,
+  isOpen,
+  currentImageIndex,
+  setIsOpen,
+  setCurrentImageIndex,
+}: StaticLightboxMobileProps) {
+  const title = useInstanceData().strings.content.imageGalleryLightboxSrTitle
+
+  function closeModal(isOpen: boolean) {
+    if (isOpen) return
+    setCurrentImageIndex(null)
+    setIsOpen(false)
+  }
+
+  function handleScroll(event: UIEvent<HTMLDivElement>) {
+    if (event.currentTarget.firstElementChild?.clientWidth === undefined) return
+    const newIndex =
+      event.currentTarget.scrollLeft /
+      event.currentTarget.firstElementChild.clientWidth
+    if (newIndex % 1 !== 0) return
+    setCurrentImageIndex(newIndex)
+  }
+
+  function handlePreviousButtonClick() {
+    if (currentImageIndex === 0) return
+    setCurrentImageIndex(currentImageIndex - 1)
+  }
+
+  function handleNextButtonClick() {
+    if (currentImageIndex === images.length - 1) return
+    setCurrentImageIndex(currentImageIndex + 1)
```

Comment:
Since we use the scrollable version with snapping we could also use scrollTo with smooth and get a nice animation for free I think.